Testing/SystemTests/tests/qt/AutomatedUITest/EngineeringDiffraction/EngDiffGuiTextureTest.py

```
@@ -0,0 +1,346 @@
+            self.assertEqual(len(RUNS), self.table().rowCount())
+
+    def _check_loading_parameters(self):
+        with self.check("Test 12 / with no parameter tables only the projection option is offered"):
+            self.assertFalse(self.texture_view.combo_param.isVisible())
+
+        self.load_parameters()
+        with self.check("Test 12 / each run is paired with its own fit parameter table"):
+            params = table_column(self.table(), COL_PARAMS)
+            self.assertNotIn("Not set", params, "a run was left without parameters")
+            # the pairing is by run number, so each row's parameter table must name the same run
+            for run_name, param_name in zip(table_column(self.table(), COL_RUN), params):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)